websites/views.py

```diff
@@ -256,10 +247,13 @@ class WebsiteContentViewSet(
     def get_queryset(self):
         parent_lookup_website = self.kwargs.get("parent_lookup_website")
         filter_type = self.request.query_params.get("type")
+        filter_subtype = self.request.query_params.get("subtype")
 
         queryset = WebsiteContent.objects.filter(website__name=parent_lookup_website)
         if filter_type:
             queryset = queryset.filter(type=filter_type)
+        if filter_subtype:
+            queryset = queryset.filter(subtype=filter_subtype)
```

Comment:
Can you add a test for this? It looks like it's not covered by code coverage https://app.codecov.io/gh/mitodl/ocw-studio/compare/172/diff#D3L256